### ISSUE TITLE
Latest version throwing syntax errors on startup

### ISSUE THREAD

**Issue by screwzloose** [essandess/isp-data-pollution#32]
```
user@ubuntu:~/isp-data-pollution$ sudo python3 ./isp_data_pollution_revised.py 
  File "./isp_data_pollution_revised.py", line 218
    print(f'This is ISP Data Pollution 🐙💨, Version {__version__}')

SyntaxError: invalid syntax
```
Is python3.6 needed to run string-formatting paradigms?

**Comment by essandess** [comment]
Yes, and possibly more importantly for `chromedriver` compatibility, though I haven't checked.

On other fronts I'm being pushed to move to 3.7.

**Comment by screwzloose** [comment]
Thanks, after installing 3.6 I was up and running!

Quick question: What does "timeout" and "short_timeout" change? Also, is there a way to randomize the request interval? e.g. Make a HTTP request, waits 2 min, requests, waits 5 min, etc?

**Comment by essandess** [comment]
`timeout` and short_timeout` control how long the main process waits for a browser child process response before killing its child and moving on. I wouldn't mess with these unless you understand the specifics or like to watch a zillion headless Chrome browsers launched in the background.

The time differences between the requests are already randomized. Grep for the `.chi2_mean_std` method. The current parameters are hardcoded in specific calls roughly based on average browsing habits.

**Comment by screwzloose** [comment]
Outstanding, thank you sir.
